R/exportToJson.R

```diff
@@ -619,9 +621,18 @@ generateDomainMetaReport <- function(conn, dbms, cdmDatabaseSchema, resultsDatab
                                             vocab_database_schema = vocabDatabaseSchema
   )  
   
-  output$MESSAGES <- querySql(conn,queryDomainMeta)
-  jsonOutput = toJSON(output)
-  write(jsonOutput, file=paste(outputPath, "/domainmeta.json", sep=""))  
+  try(
+    output$MESSAGES <- querySql(conn,queryDomainMeta)
+  )
+  if (output$MESSAGES = NULL)

```

Comment:
I've found that there is an invalid() call you can make to look for NA/null/empty value (invalid is true if any of those are the case). see: http://svitsrv25.epfl.ch/R-doc/library/gtools/html/invalid.html